src/DependencyInjection/Configuration.php

```diff
@@ -20,21 +20,21 @@ final class Configuration implements ConfigurationInterface
 {
     public const DEFAULT_LOGGER = 'logger';
 
-    public const DEFAULT_PAYMENT_METHODS = [
-        'scheme', 'dotpay', 'ideal', 'alipay', 'applepay', 'blik', 'amazonpay', 'sepadirectdebit',
-    ];
-
     public function getConfigTreeBuilder(): TreeBuilder
     {
         $treeBuilder = new TreeBuilder('sylius_adyen');
 
         $treeBuilder
             ->getRootNode()
             ->children()
-                ->arrayNode('supported_types')
```

Comment:
You need to remove also the config from `tests/TestApplication/config/config.yaml`